gemfiles/driver_oldstable.gemfile

```diff
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
-gem 'mongo', git: "https://github.com/mongodb/mongo-ruby-driver", branch: '2.15-stable'
+gem 'mongo', git: "https://github.com/mongodb/mongo-ruby-driver", branch: 'master'
```

Comment:
These would point to 2.18-stable once that branch exists.

Reply:
done